packages/common-all/src/types/store.ts

```diff
@@ -16,6 +16,7 @@ export type FindNoteOpts = {
   // If vault is provided, filter results so that only notes with matching vault is returned
   vault?: DVault;
   // If true, exclude stubs from results. Otherwise, include stub notes
+  // WARNING: If false and no other parameters are set, this will return all notes in the engine
```

Comment:
I think we need to be very careful about exposing this functionality of getting all notes, or else it'll start getting used and we'd be kinda back to engine v2.

Reply:
Yea, unfortunately there are some use cases that require all the notes (or requires a filter that we don't support). But good reminder on the `findNotesMeta`. I'll fix a few instances here and do a pass converting from NoteProps to NotePropsMeta where feasible